docs/synapse/devopsguide.rst

```diff
@@ -924,8 +924,466 @@ Kubernetes
 A popular option for Orchestration is Kubernetes. Kubernetes is an open-source system for automating the deployment,
 scaling and management of containerized applications. Synapse does work in Kubernetes environments.
 
-We are in the process of updating our Kubernetes related documentation. If you need assistance with deploying to
-a Kubernetes environment, please reach out to us directly via :ref:`synapse-support`.
+.. note::
+
+    If you are using these examples to get started with Synapse on Kubernetes, you may need to adapt them to meet
+    operational needs for your environment.
+
+.. _orch-kubernetes-deployment:
+
+Example Deployment
+~~~~~~~~~~~~~~~~~~
+
+The following deployment walks through deploying an example Synapse deployment ( based on :ref:`deploymentguide` ), but
+inside of a managed Kubernetes cluster managed by Digital Ocean. This deployment makes a few assumptions:
+
+  Synapse Deployment Guide
+    This guide assumes a familiarity with the Synapse deployment guide. Concepts covered there are not repeated here.
+
+  namespace
+    These examples uses the Kubernetes ``default`` namespace.
+
+  PersistentVolumeClaim
+    These examples use PersistetVolumeClaim (PVC) to create a persistent storage location. All Synaspe services assume
+    they have some persistent storage to read and write too.  This example uses the ``storageClass`` of
+    ``do-block-storage``. You may need to alter these examples to provide a ``storageClass`` that is appropriate
+    for your environment.
+
+  Aha naming
+    In Kubernetes, we rely on the default naming behavior for services to find the Aha service via DNS, our Aha name
```

Comment:
Feels like this sentence is missing a word, maybe a "so" like "DNS, so our Aha name"